dbtest.js

```diff
@@ -0,0 +1,17 @@
+const Pool = require("pg").Pool;
```

Comment:
Let's delete this file it's not actually doing anything for the project. It was just to test/demonstrate the db connection.